### PR TITLE
[#143, 144] Fixed two bugs in samples

### PR DESCRIPTION
**Pull request by pobudzey**
Fixes two bugs: 

- https://github.com/bean-pod/switchboard/issues/143
- https://github.com/bean-pod/switchboard/issues/144

### CODE REVIEW

File: src/receiver-ui.py [hunk 1]
```diff
@@ -26,19 +26,22 @@ def receive():
...
                     [
                         "ffplay",
                         "-autoexit",
                         "-v",
                         "warning",
-                        "-stats",
                         f"srt://{ip}:{port}?mode=listener",
                     ]
                 )
+            receiver.completed_streams.append(stream_id)
```
Comment (FGRCL) [inline src/receiver-ui.py:44]: I'm curious if this allows multiple concurrent streams, do you know if `subprocess.Popen` is non-blocking?

**Review by FGRCL — APPROVED** [review]
I only have a single questions, everything looks good to me otherwise.

File: src/receiver-ui.py (hunk 1 above)
Comment (felixlapierre) [reply src/receiver-ui.py:44]: It's definitely non-blocking, we had issues before where it was opening threads in an infinite loop and lagging my computer hard.

**Review by felixlapierre — APPROVED** [review]
Tested it out on my end and both bugs are fixed, approving

**Pull request merged**